### ISSUE TITLE
Feature request-add ability to disable insecure http

### ISSUE THREAD

**Issue by gamer191** [OnionBrowser/iOS-OnionBrowser#440]
In my opinion this should be the default, given unencrypted http leaks all your traffic to exit nodes, making it arguably less private than regular browsing (I personally would trust my ISP more than a random exit node)

Also, I find it crazy that mixed-content is enabled by default on the bronze security level, because afaik it is disabled by default on all web browsers, and I'd expect the bronze level to be somewhat equal to a regular web browser

**Comment by gamer191** [comment]
Apologies, I forgot my iOS 14 device is running an old version of Onion Browser. I'm gonna keep this issue open, because I'm unsure if the latest version of Onion Browser allows mixed-content by default. Feel free to close it

**Comment by tladesignz** [comment]
> Apologies, I forgot my iOS 14 device is running an old version of Onion Browser. I'm gonna keep this issue open, because I'm unsure if the latest version of Onion Browser allows mixed-content by default. Feel free to close it

You're right, version 3 actually doesn't allow mixed content anymore, so we're good there.

This,

> In my opinion this should be the default, given unencrypted http leaks all your traffic to exit nodes, making it arguably less private than regular browsing (I personally would trust my ISP more than a random exit node)

however, is a very valid point and a good idea!

This will get on my bucket list.
